src/settings.py

```diff
@@ -132,12 +132,13 @@ class GlobalSettings:
     grammar_search_pred_complexity_weight = 1
     grammar_search_max_predicates = 50
     grammar_search_predicate_cost_upper_bound = 6
-    grammar_search_score_function = "task_planning"
+    grammar_search_score_function = "hff_lookahead_depth0"
```

Comment:
for some reason we changed this a week ago. not sure why. changing it back to the best setting now.